packages/aws-cdk/lib/api/cloudformation-deployments.ts

```diff
@@ -1,5 +1,8 @@
+import * as path from 'path';
 import * as cxapi from '@aws-cdk/cx-api';
 import { AssetManifest } from 'cdk-assets';
+import * as fs from 'fs-extra';
+import { LazyListStackResources, ListStackResources } from '../api/evaluate-cloudformation-template';
```

Comment:
Isn't this path weird, considering this file is also in `aws-cdk/lib/api`?

Why not `from './evaluate-cloudformation-template';`?